packages/suir-component-mapper/src/files/select.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import CommonSelect from '@data-driven-forms/common/src/select';
+import FormFieldGrid from '../common/form-field-grid';
+import { validationError } from '../common/helpers';
+import { meta, input } from '@data-driven-forms/common/src/prop-types-templates';
+import { useFieldApi } from '@data-driven-forms/react-form-renderer';
+import { Dropdown } from 'semantic-ui-react';
+import FormField from '../common/form-field';
+
+const SuirSelect = ({
+  onChange,
+  value,
+  meta,
+  helperText,
+  validateOnMount,
+  isReadOnly,
+  isDisabled,
+  isClearable,
+  isSearchable,
+  options,
+  label,
+  isMulti,
+  isRequired,
+  classNamePrefix,
+  onInputChange,
+  isFetching,
+  noOptionsMessage,
+  hideSelectedOptions,
+  closeMenuOnSelect,
+  ...rest
+}) => {
+  const invalid = validationError(meta, validateOnMount);
+  let internalValue = value;
+  if (isMulti && Array.isArray(internalValue)) {
+    internalValue = value.map((item) => (typeof item === 'object' ? item.value : item));
+  }
+
+  if (!isMulti && Array.isArray(internalValue) && internalValue[0]) {
+    internalValue = typeof internalValue[0] === 'object' ? internalValue[0].value : internalValue[0];
+  }
+
+  if (!isMulti && Array.isArray(internalValue) && !internalValue[0]) {
+    internalValue = undefined;
+  }
+
+  return (
+    <FormFieldGrid helperText={helperText}>
+      <FormField
+        disabled={isDisabled}
+        readOnly={isReadOnly}
+        required={isRequired}
+        clearable={isClearable}
+        search={isSearchable}
+        loading={isFetching}
+        noResultsMessage={isFetching ? 'Loading' : noOptionsMessage()}
```

Comment:
Hardcoded loading message

Reply:
There is customization for SUIR atm. I want to address it as a part of https://github.com/data-driven-forms/react-forms/issues/484